src/cpp/desktop/DesktopMenuCallback.cpp

```diff
@@ -202,8 +201,10 @@ QAction* MenuCallback::duplicateAppMenuAction(QString commandToDuplicate,
       menuStack_.top()->addAction(pAction);
 
       auto* pBinder = new MenuActionBinder(menuStack_.top(), pAction);
-      connect(pBinder, SIGNAL(manageCommand(QString, QAction * )), this, SIGNAL(manageCommand(QString, QAction * )));
+      connect(pBinder, SIGNAL(manageCommand(QString, QAction*)),
+              this, SIGNAL(manageCommand(QString, QAction*)));
       connect(pAction, SIGNAL(triggered()), this, SLOT(actionInvoked()));
+      binders_[commandId].push_back(QPointer<MenuActionBinder>(pBinder));
```

Comment:
```suggestion
      binders_[commandId].push_back(pBinder);
```
Nit: I don't think you need to cast here